### DISCUSSION TITLE
Error while running OSMT in local: cannot match any routes

### DISCUSSION THREAD

**Discussion by UzmaParveenKhan** [wgu-opensource/osmt#196] [Asking for Help] [answered]
`polyfills.088a29ea0c66eee77c25.js:1          GET http://localhost:8080/whitelabel/whitelabel.json 404`

`main.3876e0d88b79e974c3e6.js:1 ERROR Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'oauth2/authorization/okta'
Error: Cannot match any routes. URL Segment: 'oauth2/authorization/okta'`

I can see these errors in console. can anyone please help.

**Comment by JohnKallies** [comment, marked as answer]
Hello. I'm happy to try to help. There's not a lot of context here for your error message. Are you running a Quickstart docker-compose stack for eval? Are you running the Spring app via Maven? 

Just for some base info, would you please run the osmt_sli.sh script from your local git repo? I want to understand a little about the environment running your OSMT. Please paste this command's output in a reply here:

```./osmt_cli.sh -v```

There is nothing sensitive or private in this, it simply attempts to report on the host OS and Docker/Java/Maven/NodeJS/npm environment and runtimes. 

Another community member is experiencing a similar error, but I can't speak to the root cause. That thread may be interesting to you -- https://github.com/wgu-opensource/osmt/discussions/197
